install/Knative-with-Minishift.md

```diff
@@ -111,34 +136,28 @@ bash <(curl -s https://raw.githubusercontent.com/knative/docs/master/install/scr
 
     ```shell
     curl -L https://storage.googleapis.com/knative-releases/serving/latest/istio.yaml \
```

Comment:
Any particular reason to keep the curl in here? Might be cleaner to pass the URL to `oc apply -f `

Reply:
@jcrossley3  you mean like `oc apply -f  https://storage.googleapis.com/knative-releases/serving/latest/istio.yaml` ?